File: Bubble Sort/Bubble Sort.py
def BubbleSort(array):
    n = len(array)
    for k in range(0,(n-1)):
        for i in range(0, (n-k-1)):
            if (array[i] > array[i+1]):
                array[i], array[i+1] = array[i+1], array[i]
            else:
                pass
    return a

a = [2,8,1,3,5,7,6]
T = BubbleSort(a)
print(T)

File: Bubble Sort/BubbleSort.cpp
#include<iostream>

using namespace std;
int BubbleSort(int);
int main()
{
    int a[7] = {2,8,1,3,5,7,6};
    cout<<BubbleSort(a[7]);
}

int BubbleSort(int array[7])
{
    int n = sizeof(array);
    for (int k = 0; k < (n-1); k++)
    {
        for (int i = 0; i < (n-k-1); i++)
        {
            if (array[i] > array[i+1])
            {
                array[i], array[i+1] = array[i+1], array[i];
            }
            
        }
        
    }
    return array[7];
}